app/assets/stylesheets/modules/_nav.styl

```diff
@@ -239,6 +239,12 @@ nav
         line-height 55px
         overflow hidden
         -webkit-font-smoothing auto
+    ^[2] nav
```

Comment:
What does this do? I've never seen that kind of thing is a .styl file before.

Reply:
It's a partial reference. [http://stylus-lang.com/docs/selectors.html#partial-reference](url). I took it out though. We don't need it because it's easier to adjust the selector's indention.